docs/csharp/language-reference/compiler-options/errors-warnings.md

```diff
@@ -53,6 +53,10 @@ To get information about an error or warning, you can look up the error code in
 
 ## Analysis level
 
+```xml
+<AnalysisLevel>preview</AnalysisLevel>
+```
```

Comment:
Ideally there'd be a sentence between the heading and the XML block explaining what this value controls, as is done for sibling sections. I'm not confident enough to write such a sentence however.

Reply:
I added a suggestion above. LMK what you think.